### PR TITLE
Add-ons: show warning on insufficient permissions

### PR DESCRIPTION
**Pull request by markotoplak**
##### Issue
Installing add-ons as a normal user raises PermissionErrors on Windows 

##### Description of changes
Removed user_install option (pip's --user) because:
- it does not work with conda installers
- possible clashes between different installers
- the user_install check did not work in some cases (system-wide Win7
  conda install)

Now Orange checks if it can write in site-packages folder. If it can not, it shows a warning instead of the add-on dialog.

##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by kernc** [comment]
👍 on test and warning. 👎 on removed `--user`. Now @lanzagar can no longer install add-ons via the manager.

**Comment by astaric** [comment]
He can, if he manages to run orange as root

File: Orange/canvas/application/canvasmain.py [hunk 1]
```diff
@@ -1595,7 +1595,13 @@ def open_canvas_settings(self):
             self.__update_from_settings()
 
     def open_addons(self):
-        from .addons import AddonManagerDialog
+        from .addons import AddonManagerDialog, have_install_permissions
+        if not have_install_permissions():
+            QMessageBox(QMessageBox.Warning,
+                        "Add-ons: insufficient permissions",
+                        "Insufficient permissions to install add-ons. Try starting Orange" +\
```
Comment (lanzagar) [inline Orange/canvas/application/canvasmain.py]: The "+" is not needed, but a space in one of the two strings is!

**Comment by lanzagar** [comment]
I wouldn't want to run Orange as root, but adding a parameter in Settings is acceptable :)

**Pull request merged**